weblate/trans/vcs.py

```diff
@@ -733,7 +733,7 @@ def push(self):
         on fork and creates pull request against original repository.
         """
         self.fork()
-        fork_branch = '{0}-weblate'.format(self.branch)
+        fork_branch = '{0}-weblate'.format(self.name)
```

Comment:
The problem is that with this code you always get ``GitHub-weblate`` as branch name as you are using name of VCS class, not name of the component :-).